crates/core/src/idx/ft/mod.rs

```diff
@@ -1,12 +1,12 @@
 pub(crate) mod analyzer;
 pub(crate) mod fulltext;
 pub(crate) mod highlighter;
-mod offset;
+pub(crate) mod offset;
 pub(crate) mod search;
 
-pub(super) type Position = u32;
-pub(super) type DocLength = u64;
-pub(super) type TermFrequency = u64;
-pub(super) type Score = f32;
+pub(crate) type Position = u32;
+pub(crate) type DocLength = u64;
+pub(crate) type TermFrequency = u64;
+pub(crate) type Score = f32;
```

Comment:
Why are we changing the visibility?

Reply:
`TermFrequency` and `TermFrequency` are used as values in the `key` module. I just bulk changed all of these, but I can change `Position` and `Score` back to `pub(super)` since they're not used in the `key` module.